client-tez/src/main/java/org/apache/tez/dag/app/RssDAGAppMaster.java

```diff
@@ -376,33 +399,6 @@ static void mayCloseTezSlowStart(Configuration conf) {
     }
   }
 
-  static class RssDAGAppMasterShutdownHook implements Runnable {
```

Comment:
Why do we remove this `ShutdownHook`?

Reply:
The code logic in this part has been moved to the `serviceStop` method, and it does not need to be executed repeatedly. Therefore, I have deleted it.

Reply:
> The code logic in this part has been moved to the `serviceStop` method, and it does not need to be executed repeatedly. Therefore, I remove this `ShutdownHook`.

If RssDAGAppMaster exit accidently, it  will only execute the logic of `ShutdownHook`,  it won't execute the logic of `serviceStop`, it may cause the hang of the JVM.